docs/dictionary-data.md

```diff
@@ -1,76 +1,120 @@
 The data which is needed for setting up a dictionary application is as
-follows.
-
-This is an abstract description of what’s needed, separate from details of
-field names or data formats like JSON or XML.
+follows. Note that this is an abstract description of what’s needed,
+separate from details of field names or data formats like JSON or XML.
 
 ## Language level
 
-There are some things we need to describe the language itself, before we
-look at any one particular dictionary entry.
+Some thing are needed to describe the language itself, before dealing with
+any one particular dictionary entry.
 
   - For each inflectional category:
+
     - The category name, e.g., `VAI-1`
-    - A textual description spelling out any abbreviations in the name,
```

Comment:
These (English and Cree) prose descriptions are currently provided in the relabeling scheme (`crk.atllabel.tsv`).